tests/functional/test_package.py

```diff
@@ -162,9 +163,17 @@ def osutils():
 
 
 @pytest.fixture
-def pip_runner():
+def empty_env_osutils():
+    class EmptyEnv(object):
+        def environ(self):
+            return {}
+    return EmptyEnv()
+
+
+@pytest.fixture
+def pip_runner(empty_env_osutils):
```

Comment:
Do we need the `pip_runner` fixture anymore after the refactor?

Reply:
What's the alternative?  The `pip_runner` fixture is used in numerous tests still.  I'm not following.

Reply:
I was referring to the fact that he switched out the use of `pip_runner` with `pip_factory`. I was not sure if the `pip_runner` fixture was being used anymore. When I did a quick check, I did not see it being used anywhere else in the `test_package.py` file

Reply:
Almost all the tests in `TestDependencyBuilder` in the class below use this.  Am I missing something?

Reply:
Nevermind. I was looking at the `tests/unit/deploy/test_packager.py` file when I was going over the file. The `tests/functional/test_package.py` definitely needs it. Did not help that the same fixture was duplicated in two different test files instead of having it in the `conftest.py`.